src/Tmds.Ssh/SftpClient.File.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Buffers;
+using System.Buffers.Binary;
+
+namespace Tmds.Ssh
+{
+    public class SftpFile
+    {
+        private byte[] handle;
+        internal SftpFile(byte[] handle)
+        {
+            this.handle = handle;
+        }
+    }
```

Comment:
To make it a bit more interesting, let's also implement the `ValueTask CloseAsync();` method on this type.

Reply:
Hmm, should the method actually send requests to close the handle? How should that be implemented outside of the SftpClient?

Reply:
I've implemented something for now

Reply:
We don't want `SftpClient` as an argument here, can you make it a field that gets set in the constructor?